spec/factories/posts.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :post do
+    user_id 1
+content "MyString"
+  end
+
+end
```

Comment:
But now bring this puppy into your tests. Also, how something like this usually ends up looking like:
```
FactoryGirl.define do
  factory :post do
    user_id 1
    content { Faker::Company.buzzword }
  end
end
```